### ISSUE TITLE
dropout error

### ISSUE THREAD

**Issue by iamownt** [joosthub/PyTorchNLPBook#28]
In fact, F.dropout(xx, p=0.5) works both in model's train mode and eval mode. You should write F.dropout(xx,  p=0.5, training=self.training) or when you excute a cell more than one time, the result is different. I heard this book from the CS224 class, the professor said that it's a great book and if we don't  read the book, he will be sad. Even now there seems no one maintaining this book. I want to say "Thank you, Great authors!"
